drivers/clock_control/clock_control_wch_rcc.c

```diff
@@ -34,6 +34,12 @@
 #define WCH_RCC_SRC_IS_HSI 1
 #endif
 
+#if DT_NODE_HAS_COMPAT_STATUS(DT_NODELABEL(clk_lse), wch_ch32v00x_hse_clock, okay)
+#define WCH_HSE_BYPASS DT_PROP(DT_NODELABEL(clk_lse), lse_bypass)
+#elif
+#define WCH_HSE_BYPASS 0
+#endif
```

Comment:
I wonder if this would work with just [`DT_PROP_OR()`](https://docs.zephyrproject.org/latest/doxygen/html/group__devicetree-generic-prop.html#ga5e5bfc9b1a6627b3f73014329e96340f) and no `DT_NODE_HAS_...`

```c
#define WCH_HSE_BYPASS DT_PROP_OR(DT_NODELABEL(clk_lse), lse_bypass, 0)
```

But I am not sure this does what you wish though. Just for your own convenience.

Reply:
same, just learning them while writing those PRs :)